src/components/Terminal.jsx

```diff
@@ -140,14 +139,10 @@ export default class Terminal extends React.Component {
   }
 
   getStdout () {
-    const lineStyle = {
```

Comment:
Why is `lineStyle` removed here, yet no new default style provided anywhere as far as I can see?

Reply:
Like I wrote above.
Classname-based style could be modified by both css and js, but inlineStyles can only be modified by js.
And, inlineStyles have higher priority than classname, people maynot change is as they want.